brigade/static/scss/molecules/_message.scss

```diff
@@ -5,14 +5,21 @@
 //
 
 .message {
-  background-color: $color-gray-light;
-  border: 0;
-  border-radius: 3px;
+  background-color: $color-white;
+  border: solid $color-gray-light 1px;
```

Comment:
I think this is a typo - should be `1px solid $color-gray-light`.

Reply:
Good catch